requirements.txt

```diff
@@ -59,3 +59,6 @@ urllib3==1.22             # via requests
 vine==1.1.4               # via amqp
 xmltodict==0.11.0         # via anymarkup
 radon==2.0.0
+
+# Celery debugging
+watchdog
```

Comment:
just please add the `watchdog` package into `requirements.in` and then regenerate the `requirements.txt`

Reply:
Thanks @tisnik, will do.